### ISSUE TITLE
Confirmation msg on login

### ISSUE THREAD

**Issue by Ishika-Gupta06** [Vin205/Enyanjyoti#197]
@Vin205 assign this to me under GSSOC-EXT, hacktoberfest, level-1,2,3

**Comment by VanshKirtishahi** [comment]
Is there an existing issue for this?
I have searched the existing issues and found issue #197 regarding the confirmation message on login.

Feature Description:
I would like to implement a confirmation message that appears after a user successfully logs in to the application.

Use Case:
This feature will inform users that they have successfully logged in, enhancing the user experience.

Benefits:

Provides immediate feedback to the user.
Helps prevent confusion and improves usability.
Add Screenshots:
No screenshots are available at this time, but I can provide mockups if needed.

Priority:
I consider this a high priority feature, as it directly affects user experience.

Record:
I have read the Contributing Guidelines.
I am a GSSOC'24 contributor.
I have starred the repository.

Thank you for considering my request! I look forward to your feedback.
